src/dbSta/include/db_sta/dbNetwork.hh

```diff
@@ -253,7 +253,7 @@ class dbNetwork : public ConcreteNetwork
   void findConstantNets();
   virtual void visitConnectedPins(const Net* net,
                                   PinVisitor& visitor,
-                                  ConstNetSet& visited_nets) const;
+                                  NetSet& visited_nets) const;
```

Comment:
This no longer overrides the base class method and therefore seems wrong.  What is the point of this change?

Reply:
The ConstNetSet class is no longer exist, and seems it's renamed to NetSet. So, without these changes, build will be failed.

Reply:
It seems like the STA version is bumped since the fixes are similar to #2938

Reply:
@gadfort Yes. It was fixed in this PR and mine. When these changes are updated to the master branch, my issue will be solved.